BossMod/Replay/Visualization/ReplayDetailsWindow.cs

```diff
@@ -40,7 +40,7 @@ public DateTime CurrentTime
         _player = new(data);
         _rotationDB = rotationDB;
         _mgr = new(_player.WorldState);
-        _hintsBuilder = new(_player.WorldState, _mgr);
+        _hintsBuilder = new(_player.WorldState, _mgr, new(_player.WorldState, _mgr));
```

Comment:
Who will dispose questbattledirector here? I think it should be a field in this class, similar to bossmodulemanager.

Reply:
moved